panda/src/gobj/texture.h

```diff
@@ -749,6 +749,7 @@ class EXPCL_PANDA_GOBJ Texture : public TypedWritableReferenceCount, public Nama
   bool do_has_compression(const CData *cdata) const;
   virtual bool do_has_ram_image(const CData *cdata) const;
   virtual bool do_has_uncompressed_ram_image(const CData *cdata) const;
+
```

Comment:
Since this is the only change to this file and it's just a whitespace change, I would prefer to see this change omitted from the PR to keep it smaller.